### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kojix2**
Motivation : #10
 
I think this package is useful to the Ruby community, so it should be a little more discoverable across the planet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
-# README
+# Native Package Installer
```
Comment (kou) [inline README.md]: Could you use "native-package-installer"?

We don't have the label of this package yet. So we need to use the package ID for now.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
-# README
+# Native Package Installer 
 
-TODO...
+[![Gem Version](https://badge.fury.io/rb/native-package-installer.svg)](https://badge.fury.io/rb/native-package-installer)
+
+Native Package Installer let you install native packages with `gem install` command.
+
+You can install both native packages and C extensions by one action.
```
Comment (kou) [inline README.md]: Could you use the description in gemspec?
https://github.com/ruby-gnome/native-package-installer/blob/master/native-package-installer.gemspec#L28-L34

File: README.md (hunk 1 above)
Comment (kojix2) [reply README.md]: Thanks. done.

File: README.md (hunk 2 above)
Comment (kojix2) [reply README.md]: Thanks. done.

**Pull request merged**